### ISSUE TITLE
dssm model gragh.py  forward（） logits = tf.concat([pos_result, neg_result], axis=1) there why pos_result put forward? Is wrong? 

### ISSUE THREAD

**Issue by yangyan1227** [terrifyzhao/text_matching#6]

**Comment by yangyan1227** [comment]
dssm model gragh.py  forward（） logits = tf.concat([pos_result, neg_result], axis=1)
there why pos_result put forward? Is wrong?
